src/nodes/functions/BasicLightingModel.js

```diff
@@ -12,9 +12,11 @@ class BasicLightingModel extends LightingModel {
 
 	}
 
-	indirectDiffuse( { reflectedLight } ) {
+	indirectDiffuse( { ambientOcclusion, reflectedLight } ) {
 
-		reflectedLight.indirectDiffuse.assign( diffuseColor.rgb );
+		reflectedLight.indirectDiffuse.addAssign( diffuseColor.rgb );
+
+		reflectedLight.indirectDiffuse.mulAssign( ambientOcclusion );
```

Comment:
I was thinking in we move this to `LighthingModel.ambientOcclusion()`, since we already have this function.

Reply:
Um, maybe it is easier to read the code when the final computation of indirect diffuse (accumulation + modulation) happens at one place. 

Although when the line is moved to `LighthingModel.ambientOcclusion()` we don't have to repeat it in derived models. I'm just a bit afraid it will make it more hard to understand the lighting models in the end...